site2/docs/how-to-landing.md

```diff
@@ -0,0 +1,16 @@
+---
+Id: how-to-landing
+title: How-to
+sidebar_label: “”
+—
+
+Learning new software can be an overwhelming task, but relax – most aspects of Pulsar can be easily configured in just a few steps. These tutorials will show you how to quickly create topics, tenants, and namespaces, produce and consume messages, and more!

```

Comment:
```suggestion
Learning new software can be an overwhelming task, but relax – most aspects of Pulsar can be easily configured in just a few steps. These tutorials show you how to quickly create topics, tenants, and namespaces, produce and consume messages, and more!
```

Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Reply:
OK